protos/feast/core/Store.proto

```diff
@@ -36,13 +35,17 @@ message ServingStore {
   string name = 1;
 
   // Type of store.
-  ServingStore.StoreType type = 2;
+  Store.StoreType type = 2;
 
   // URI to connect to the database. Should contain all necessary information
   // to connect to and write/read to the store.
   string databaseURI = 3;
 
   // Feature sets to subscribe to. Regex strings are allowed, in the form of
-  // featureSetId:version. E.g. my-feature-set:2^
+  // featureSetId:version.
+  // Valid options for version:
+  //     latest: only subscribe to latest version of feature set
+  //     [version number]: pin to a specific version
+  //     [version number]^: subscribe to all versions larger than or equal to [version number]
   repeated string subscriptions = 4;
```

Comment:
How are subscriptions for specific stores updated? How are stores added?

Reply:
This is yet to be decided. For now we are just capturing the type information